### PR TITLE
code change: Fix issues found by inspection tool

### PR DESCRIPTION
**Pull request by akrejcir**
**What this PR does / why we need it**:
Fixed some issues found by "Inspect code" tool in Goland.

**Release note**:
```release-note
None
```

### CODE REVIEW

File: internal/common/scheme.go [hunk 1]
```diff
@@ -19,5 +19,5 @@ func init() {
 	utilruntime.Must(clientgoscheme.AddToScheme(Scheme))
 	utilruntime.Must(extv1.AddToScheme(Scheme))
 	utilruntime.Must(sspv1beta1.AddToScheme(Scheme))
-	utilruntime.Must(osconfv1.AddToScheme(Scheme))
+	utilruntime.Must(osconfv1.Install(Scheme))
```
Comment (ksimon1) [inline internal/common/scheme.go:22]: What does this change do?

Comment (akrejcir) [reply internal/common/scheme.go:22]: `AddToScheme` is deprecated, `Install` is the same function, with different name:
https://github.com/openshift/api/blob/master/config/v1/register.go#L21

Comment (ksimon1) [reply internal/common/scheme.go:22]: So can you please change it for all schemas?

Comment (akrejcir) [reply internal/common/scheme.go:22]: Not all modules rename it. Some keep the `AddToScheme` name. I guess they will rename it in later versions.

Comment (akrejcir) [reply internal/common/scheme.go:22]: You are right, it is only deprecated in OpenShift API.

**Pull request merged**